### DISCUSSION TITLE
Шаблонизаторы | Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#18] [Announcements]
# Шаблонизаторы | Node.js с примерами кода

Справочник web-разработчика с примерами кода - Node.js

https://nodejsdev.ru/doc/templates/

**Comment by hahavenn** [comment]
В месте первого примера
//  app.engine(
//    'handlebars',
//    handlebars({ defaultLayout: 'main' })
//  )
не вызывается метод engine модуля handlebars(объект не является функций). Надо вызвать метод данного модуля:
//  app.engine(
//  "handlebars", 
//  handlebars.engine({ defaultLayout: "main" })
//  )

**Reply by bndby** [reply]
Спасибо, поправил.
